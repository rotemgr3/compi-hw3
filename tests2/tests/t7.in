//Ex3 pdf - should be ambiguousCall +
override byte foo(int x) {return 5b;}
override byte foo(byte x) {return 5b;}
void main() {
	foo(5);
	int a = foo(2b);
}

