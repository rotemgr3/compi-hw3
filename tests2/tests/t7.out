---end scope---
x INT -1
---end scope---
x BYTE -1
line 6: ambiguous call to overloaded function foo
